### ISSUE TITLE
[BUG] [Experiment View] Insufficient color contrast for the icon on the "Add Member" button

### ISSUE THREAD

**Issue by nadzeyapiatrouskaya-boop** [epam/Indigo-ELN-v.-2.0#763]
**Steps to Reproduce:**

1. Open app: https://indigo-eln-dev.test.lifescience.opensource.epam.com/ and log in as a user with "MANAGE_DICTIONARIES" permission (e.g. as admin)
2. Navigate to the any Experiment page
3.Open Add Team Member tab
4. Take a look at icon of Add Member button

**Actual result:**
The icon has a very low contrast ratio against the button's background (, making it barely visible to users, especially those with visual impairments.

<img width="1913" height="1025" alt="Image" src="https://github.com/user-attachments/assets/d507e99e-3acf-4053-8b29-bfe36236e65f" />

**Expected result:** 
The icon on the "Add Member" button is clearly visible and meets WCAG AA standards, having a contrast ratio of at least 3:1 against the button background.
